pyproject.toml

```diff
@@ -65,4 +65,7 @@ name = "Misc"
 showcontent = false
 
 [tool.black]
+diff = true
+color = true
+check = true
```

Comment:
I would remove `check = true` because that means `black` won't reformat my files when I run it locally. I want `black` to reformat my files when I run it locally.